request.go

```diff
@@ -88,6 +95,15 @@ func NewRequestListFilter() *RequestListFilter {
 	return &RequestListFilter{Values: url.Values{}}
 }
 
+// Clone copies the embedded url.Values over to a new RequestListFilter
```

Comment:
Don't document the internals of the external Type. Clone clones the RequestListFilter is enough